activesupport/test/key_generator_test.rb

```diff
@@ -19,7 +19,7 @@ def setup
   test "Generating a key of the default length" do
     derived_key = @generator.generate_key("some_salt")
     assert_kind_of String, derived_key
-    assert_equal OpenSSL::Digest::SHA1.new.block_length, derived_key.length, "Should have generated a key of the default size"
+    assert_equal OpenSSL::Cipher::Cipher.new('aes-256-cbc').key_len, derived_key.length, "Should have generated a key of the default size"
```

Comment:
This should be OpenSSL::Cipher.new(...). OpenSSL::Cipher::Cipher has been deprecated since Ruby 1.9.